docs/_templates_system/templates.rst

```diff
@@ -104,3 +118,17 @@ On template level every variable in Context and fetched by :code:`gimme` and :co
 
     {{ article.title }}
     {{ article['title']}}
+
+**generate url**
+
+.. code-block:: twig
+
+    {{ url(article) }}
```

Comment:
its worth to add that `path(article)` will also generate the url for provided object